spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -234,6 +234,7 @@ def assert_specific_cloud_volume
       expect(cloud_volume).to have_attributes(
         :ems_ref     => volume_data['id'].partition('/').last,
         :name        => volume_data['pi_volume_name'],
+        :description => "IBM Cloud Block-Storage Volume",
```

Comment:
These are attributes for a specific `CloudVolume` ([`volumes` parser ref](https://github.com/miyamotoh/manageiq-providers-ibm_cloud/blob/new-volume-type-dropdown/app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb#L179)) `CloudVolumeType` ([`cloud_volume_types` parser ref](https://github.com/miyamotoh/manageiq-providers-ibm_cloud/blob/new-volume-type-dropdown/app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb#L296)).

I think we need to define a new `assert_specific_cloud_volume_type` method in our refresher spec, checking the `:name` and `:description` attributes for a `CloudVolumeType` instance.

Reply:
Oh, I see. Please take a look at the latest force-push where I attempt to make sure `:name` and `:description` of a `cloud_volume_type` are non-empty.

Reply:
Perfect, thanks @miyamotoh !